src/Mint-Y/cinnamon/sass/_common.scss

```diff
@@ -1473,9 +1473,10 @@ StScrollBar {
       background-color: $noaccent_selected_bg_color;
       border: 1px solid $borders_color;
     }
-    &-hover {
-      background-color: red;
-      border-radius: 2px;
+    &:hover {
+      background-color: rgba(132,132,132,0.14);
```

Comment:
scratch that.. you want it a shade lighter than the hover-to-select mode. It could still be a named color though, or a modification of one.

Reply:
Yes, I agree. I was trying to come up with something simple that would work for both light and dark themes but I ended up using `if($variant=='light', ...` for the border anyway because I couldn't find an easy solution with the named colors. In any case, I think I've found the correct solution now that should adapt well if any of the named colors are changed in future. https://github.com/linuxmint/mint-themes/pull/433